phpcs.xml.dist

```diff
@@ -6,6 +6,20 @@
   <arg name="extensions" value="inc,install,module,php,profile,test,theme,yml"/>
 
   <rule ref="Drupal"></rule>
+  <rule ref="DrupalPractice"></rule>
+
+  <rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis.UnusedVariable">
```

Comment:
We should add a comment above "TODO: check if we can fix those reported errors in code"